app/v3/controllers/requestParsers/AmendLossClaimsOrderParser.scala

```diff
@@ -26,9 +26,8 @@ class AmendLossClaimsOrderParser @Inject()(val validator: AmendLossClaimsOrderVa
 
   override protected def requestFor(data: AmendLossClaimsOrderRawData): AmendLossClaimsOrderRequest = {
 
-    val taxYear: DownstreamTaxYear = data.taxYear match {
-      case Some(year) => DownstreamTaxYear.fromMtd(year)
-      case None => DownstreamTaxYear.mostRecentTaxYear()
+    val taxYear: DownstreamTaxYear = data.taxYearClaimedFor match {
+      case year => DownstreamTaxYear.fromMtd(year)
```

Comment:
No need for a match any more so simpler if just use:
```suggestion
    val taxYear = DownstreamTaxYear.fromMtd(data.taxYearClaimedFor)
```